routes/transactions.js

```diff
@@ -184,4 +184,27 @@ router
         }
     });
 
+    router.route('/transaction/:id').post(async (req, res) => {
+
+        if(req.query.method === 'DELETE'){
```

Comment:
What is the intended flow here? It's coming in as a POST request, but it's checking to see if it's a DELETE method, so I'm kind of confused.